### PR TITLE
Fixes #27005 - Fix Editor changing locked template

### PR DESCRIPTION
**Pull request by glekner**

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/Editor/Editor.js [hunk 1]
```diff
@@ -188,7 +188,9 @@ class Editor extends React.Component {
           isMasked={isMasked}
           isRendering={isRendering}
         />
-        <textarea className="hidden" name={name} value={value} readOnly />
+        {!readOnly && (
+          <textarea className="hidden" name={name} value={value} readOnly />
```
Comment (ezr-ondrej) [inline webpack/assets/javascripts/react_app/components/Editor/Editor.js:192]: isn't the `readOnly` prop useless now as it is always false?

**Review by ezr-ondrej — COMMENTED** [review]
:+1: Thanks @glekner, fixes the issue and LGTM.
Only one nitpick.

File: webpack/assets/javascripts/react_app/components/Editor/Editor.js (hunk 1 above)
Comment (glekner) [reply webpack/assets/javascripts/react_app/components/Editor/Editor.js:192]: its a `textarea` without `onChange` handler. due to the Editor taking care of the `onChange`, it needs to be `readOnly` aswell.

Comment (ezr-ondrej) [reply webpack/assets/javascripts/react_app/components/Editor/Editor.js:192]: ack! thanks for explanation! :heart:

**Review by ezr-ondrej — DISMISSED** [review]

**Comment by ezr-ondrej** [comment]
The errors are unrelated, but maybe would be better to wait for them to be resolved anyway. The fix is in #6833

**Review by amirfefer — DISMISSED** [review]
Tested and works well :+1: 
Thanks @glekner

**Review by ohadlevy — DISMISSED** [review]

File: webpack/assets/javascripts/react_app/components/Editor/Editor.js (hunk 1 above)
Comment (ohadlevy) [reply webpack/assets/javascripts/react_app/components/Editor/Editor.js:192]: does this warrant adding a test?

Comment (glekner) [reply webpack/assets/javascripts/react_app/components/Editor/Editor.js:192]: added a test. @ohadlevy

**Review by ohadlevy — APPROVED** [review]

**Pull request merged**